.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:2.7.15
```

Comment:
Python-2.7 is EOL in 2020.  I'd prefer new dev to be against v3 if we can please?

Reply:
I started with some code from AWS labs and set the env to Python3, but it didn't work until I put it to 2.7. I agree that there should be a follow on piece of work to make this work in Python3 though.